app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1300,7 +1299,6 @@ class PatientRepositoryAndroidTest {
   fun verify_deleted_appointments_are_not_included_when_fetching_recent_patients() {
     val facilityUuid = UUID.fromString("086a5afd-c14b-4023-8eaa-9f29eee180a5")
     val recentPatient1 = savePatientWithAppointment(facilityUuid = facilityUuid, creationFacilityUuid = facilityUuid)
-    val recentPatient2 = savePatientWithAppointment(facilityUuid = facilityUuid, creationFacilityUuid = facilityUuid, deletedAt = Instant.now())
```

Comment:
Why was this removed? This is needed because we want to ensure that deleted patients are not fetched in the recent patients query.